### PR TITLE
Create example for Markup with JSX

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Finish challenge in writing Markup with JSX

https://react.dev/learn/writing-markup-with-jsx#challenges

image

### CODE REVIEW

File: examples/JSX-example/eslint.config.mjs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// @ts-check
+
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+);
```
Comment (hungdo-agilityio) [inline examples/JSX-example/eslint.config.mjs]: Can you help to recheck this file

File: examples/JSX-example/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "@eslint/compat": "^1.0.1",
+    "@eslint/js": "^9.3.0",
+    "@types/eslint__js": "^8.42.3",
+    "eslint": "^8.57.0",
+    "eslint-plugin-react": "^7.34.2",
+    "globals": "^15.3.0",
+    "typescript": "^4.9.5",
+    "typescript-eslint": "^7.11.0"
+  }
+}
```
Comment (hungdo-agilityio) [inline examples/JSX-example/package.json]: Please check warning here

Comment (thanhtrantruong47) [reply examples/JSX-example/package.json]: Fixed

File: examples/JSX-example/eslint.config.mjs (hunk 1 above)
Comment (thanhtrantruong47) [reply examples/JSX-example/eslint.config.mjs]: Fixed

**Comment by thanhnguyen-agilityio** [comment]
Similar this [comment](https://github.com/thanhtrantruong47/react-training/pull/4#issuecomment-2139589571), `JSX-example` should be another folder. Do not update it override the current one.

The folder structure should be:
```
react training/
- - examples/
- - - - my-first-react-app/
- - - - JSX-example/
- - - - thinking-in-react/
```

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**